src/main.rs

```diff
@@ -1,16 +1,104 @@
-use anyhow::{ Result};
-use cli_lang_tool::{Dictionary, Word};
+use anyhow::{Result};
+use clli::{Dictionary, Word};
 use std::{fs::File, io::{prelude::*, BufReader}, path::PathBuf};
 use serde::{Serialize, Deserialize};
 use rand::Rng;
-use structopt::StructOpt;
+use clap::{ Parser, Subcommand};
 
 
 const FILE_PATH: &str = "./examples/storage/turk.json";
 
+#[derive(Debug, Parser)]
+#[clap(name = "clli")]
+#[clap(about = "turkish lang tool", long_about = None)]
+struct Cli {
+    #[clap(subcommand)]
+    command: Commands,
+}
+
+#[derive(Debug, Subcommand)]
+enum Commands {
+    // #[clap(arg_required_else_help = true)]
+    ///Get the random word from the sozluk
+    Random{
+        /// count - how many random words
+        #[clap(default_value_t = 1)]
+        count: u8,
+    },
+    #[clap(arg_required_else_help = true)]
+    /// Adds records to the dictionary
+    Add
+        {
+        #[clap(required = true)]
+        native: String,
+        english: String,
+        },
+
+    /// Get the count of words in the dictionary 
+    Count,
+
+    /// Get the all words from the dictionaryy
+    GetAllWords,
+
+    /// Translate english -> turkish
+    Etranslate{
```

Comment:
I wouldn't fix on E or T. If we want this solution to be generic, I believe having just one translate would be great. We could imply that most of the time people would translate things from the main language to English. The flow I see here is this:
```
clli translate sozluk
clli translate sozluk --reverse # or perhaps --en meaning that we want to do a translation from the default - en 
```